### PR TITLE
Workout data

### PR DESCRIPTION
**Pull request by talatcanoguz**

### CODE REVIEW

**Comment by talatcanoguz** [comment]
**lgtm!!!!!**

File: lib/services/firestore_repository.dart [hunk 1]
```diff
@@ -17,4 +18,37 @@ class FireStoreRepository {
...
   }
+
+  Future<List<Map<String, dynamic>>> fetchLastFiveDays(User user) async {
+    final now = DateTime.now();
+    List<Map<String, dynamic>> daysData = [];
+
+    for (int i = 0; i < 5; i++) {
+      DateTime day = now.subtract(Duration(days: i));
+
+      var dayDocSnapshot = await _firestore
```
Comment (Thomas-Chambers) [inline lib/services/firestore_repository.dart:29]: do you think there is any way that we can get around having to do 5 calls to firestore, like getting the whole collection for the month, unsure how to handle the transitions between the two months in that case... we could also do the thing where we just store the 5 most recent workouts separately from the 'bulk' workout storage

File: lib/view_models/user_view_model.dart
Comment (Thomas-Chambers) [inline lib/view_models/user_view_model.dart:1]: we could maybe rename to firestore_view_model to better reflect what it does

File: lib/services/firestore_repository.dart (hunk 1 above)
Comment (talatcanoguz) [reply lib/services/firestore_repository.dart:29]: I think storing the past 5 as a bulk is better.

**Comment by talatcanoguz** [comment]
Sadly, I have to close you, my dear friend.

**Comment by talatcanoguz** [comment]
farewell...

**Comment by Thomas-Chambers** [comment]
Talat,

I have added a 1:1 to your calendar tomorrow to discuss your recent behavior.

-Thomas

**Comment by talatcanoguz** [comment]
Thomas, you're fired.

**Comment by Thomas-Chambers** [comment]
🤙

**Pull request closed without merging**